index.html

```diff
@@ -196,16 +196,25 @@ <h2><dfn>Performance Timeline</dfn></h2>
       <li>a <dfn>performance observer task queued flag</dfn></li>
       <li>a <dfn>list of <a>registered performance observer</a> objects</dfn>
       that is initially empty</li>
-      <li>a <dfn>performance entry buffer map</dfn>, one per
-      <dfn>entryType</dfn>, to store <a>PerformanceEntry</a> objects, that
-      are initially empty.  Each entryType's buffer has a
-      <dfn>maxBufferSize</dfn> that with rare exception, cannot be changed.
+      <li>a <dfn>performance entry buffer map</dfn> that is a mapping of a
+        <code>DOMString</code>, representing an entryType supported by the
+        UserAgent, to the following tuple:
+        <ul>
+          <li> A <dfn>performance entry buffer</dfn> to store
+            <a>PerformanceEntry</a> objects, that is initially empty</li>
+          <li> A <code>long</code> indicating the <dfn>maxBufferSize</dfn> for
+            this entryType.  With rare exception, this value cannot be changed
+            once set</li>
+          <li> A <code>boolean</code> <dfn>availableFromTimeline<dfn>
+            representing whether this entryType is available from the
+            <a>Performance</a> interface.</li>
+        </ul>
       </li>
     </ul>
-    <p>The <dfn>relevant performance entry buffer</dfn> is the object in the
-    <a>performance entry buffer map</a> associated with the <a data-cite=
-    "HTML/webappapis.html#concept-relevant-global">relevant global
-    object</a> for a specific <dfn>entryType</dfn>.</p>
+    <p>The <dfn>relevant performance entry buffer</dfn> is the <a>performance
```

Comment:
This dfn now needs an entryType as a parameter in order to make sense.

Reply:
Maybe we can simplify it somehow to make it less confusing. But there is not a single performance entry buffer now. So that did not work as it was.

Reply:
resolved with Yoav's changes